spec/features/logging/view_auth_requests_for_a_username_spec.rb

```diff
@@ -42,7 +55,13 @@
           })
         .to_return(status: 200, body: logs.to_json, headers: {})
 
-      sign_in_user user
+      organisation = create(:organisation)
+      location_two = create(:location, organisation: organisation)
+
+      create(:ip, location_id: location.id, address: "1.1.1.1")
+      create(:ip, location: location_two, address: "2.2.2.2")
```

Comment:
I don't think you need any of lines 58 - 62 here. You already have an organisation (named `organisation`) from line. 6. And you don't actually use these IPs after you've created them as we are just stubbing the response to have a log with the text "2.2.2.2" we are not actually looking for objects in the db.